modules/management/src/test/java/org/mule/management/config/ManagementNamespaceHandlerTestCase.java

```diff
@@ -107,18 +107,18 @@ public void testAgentsOrder() throws Exception
         assertEquals(agents.size(), 8);
         
         Iterator<Agent> iter = agents.iterator();
-        assertTrue(iter.next() instanceof JmxApplicationAgent);
```

Comment:
the test is lousy and checks for ordering of things registered. I just went along with it